conductr_cli/validation.py

```diff
@@ -169,6 +169,24 @@ def handler(*args, **kwargs):
     return handler
 
 
+def handle_conduct_load_read_timeout_error(func):
+    def handler(*args, **kwargs):
+        try:
+            return func(*args, **kwargs)
+        except ReadTimeout as err:
+            log = get_logger_for_func(func)
+            log.error('Timed out waiting for response from the server: {}'.format(err.args[0]))
+            log.error('One possible issue may be that there are not enough resources or machines with the roles '
+                      'that your bundle requires')
```

Comment:
@huntc - error message updated; I changed `you bundle requires` to `your bundle requires`